dev_environment/setup_dev_env.sh

```diff
@@ -0,0 +1,183 @@
+      grafana \
+      libc6-dev \
+      libcurl4-openssl-dev \
+      libelf-dev \
+      libjson-c-dev \
+      libncurses5-dev \
+      libpcap-dev \
+      libssl-dev \
+      linux-headers-generic \
+      linux-tools-common \
+      linux-tools-generic \
+      llvm \
+      prometheus \
+      rsync
+
+#install go lang version 1.17.12
+if [ -d "/usr/local/go/bin" ];
+then
+  echo "golang already installed"
+else
+        if [[ "$ARCH" = "arm" ||  "$ARCH" = "aarch64" ]];
+        then
+                wget https://go.dev/dl/go1.17.12.linux-arm64.tar.gz
+                tar -C /usr/local -xzf go1.17.12.linux-arm64.tar.gz
+        elif [ "$ARCH" = "x86_64" ];
+        then
+                wget https://go.dev/dl/go1.17.12.linux-amd64.tar.gz
+                tar -C /usr/local -xzf go1.17.12.linux-amd64.tar.gz
+        fi
+fi
+export PATH=$PATH:/usr/local/go/bin
+
+# clone the l3afd repo in to root directly
+# can use mapped directory i.e. at /home/ubuntu/Home
+if [ -d "/root/l3afd" ];
+then
+  echo "repo already exited"
+else
+  git clone https://github.com/l3af-project/l3afd.git
+  git clone https://github.com/l3af-project/l3af-arch.git
+fi
+
+# Copy grafana configs and dashboards into place
+if [ -d "/var/lib/grafana/dashboards"];
+then
+  echo "grafana directory already existed"
+else
+  mkdir -p /var/lib/grafana/dashboards
+fi
+chown grafana:grafana /var/lib/grafana/dashboards
+cp /root/l3af-arch/dev_environment/cfg/grafana/dashboards/*.json /var/lib/grafana/dashboards
+chown grafana:grafana /var/lib/grafana/dashboards/*.json
+cp /root/l3af-arch/dev_environment/cfg/grafana/provisioning/dashboards/l3af.yaml /etc/grafana/provisioning/dashboards
+chown root:grafana /etc/grafana/provisioning/dashboards/*.yaml
+cp /root/l3af-arch/dev_environment/cfg/grafana/provisioning/datasources/l3af.yaml /etc/grafana/provisioning/datasources
+chown root:grafana /etc/grafana/provisioning/datasources/*.yaml
+
+# Copy prometheus config and restart prometheus
+cp /root/l3af-arch/dev_environment/cfg/prometheus.yml /etc/prometheus/prometheus.yml
+if uname -a | grep -q 'WSL'; then
+        echo "WSL DETECTED"
+        apt-get install daemon
+        #start/restart prometheus-node-exporter
+        /etc/init.d/prometheus-node-exporter start
+        /etc/init.d/prometheus-node-exporter stop
+        /etc/init.d/prometheus-node-exporter start
+
+        # Start and enable Grafana
+        sleep 1
+        /etc/init.d/grafana-server stop || true
+        /etc/init.d/grafana-server stop || true
+        /etc/init.d/grafana-server start || true
+else
+  systemctl daemon-reload
+  systemctl restart prometheus prometheus-node-exporter
+
+  # Start and enable Grafana
+  systemctl daemon-reload
+  systemctl start grafana-server
+  systemctl enable grafana-server.service
+fi
+
+# Get Linux source code to build our eBPF programs against
+if [ ! -d "/usr/src/linux" ];
+then
+  git clone --branch v5.1 --depth 1 https://github.com/torvalds/linux.git /usr/src/linux
```

Comment:
As discussed during the weekly TSC meeting, we are still going to build from the source.  When we changes on the main line to install the package, this will follow.